### PR TITLE
Update Js documentation for customers pages

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Update Js documentation for customers pages
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24660)

### CODE REVIEW

File: tests/UI/pages/BO/customers/index.js [hunk 1]
```diff
@@ -270,8 +285,8 @@ class Customers extends BOBasePage {
 
   /**
    * Get content from all rows
-   * @param page
-   * @param column
+   * @param page {Page} Browser tab
+   * @param column {string} Column nam to get all rows content
```
Comment (matks) [inline tests/UI/pages/BO/customers/index.js]:
```suggestion
   * @param column {string} Column name to get all rows content
```

**Review by matks — COMMENTED** [review]
Seems OK, I might have found a little mistake? 😉

File: tests/UI/pages/BO/customers/view.js [hunk 2]
```diff
@@ -60,8 +69,8 @@ class ViewCustomer extends BOBasePage {
 
   /**
    * Get number of element from title
-   * @param page
-   * @param cardTitle
+   * @param page {Page} Browser tab
+   * @param cardTitle {string} Value of card title to get number of element
```
Comment (Progi1984) [inline tests/UI/pages/BO/customers/view.js]:
```suggestion
   * @param cardTitle {string} Value of card title to get number of elements
```

**Review by Progi1984 — COMMENTED** [review]
Some wording feedback

File: tests/UI/pages/BO/customers/view.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/customers/view.js]: Thx done :+1:

File: tests/UI/pages/BO/customers/index.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/customers/index.js]: Done thx :+1:

File: themes/core.js.LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,54 @@
+/*!
```
Comment (boubkerbribri) [inline themes/core.js.LICENSE.txt]: Please delete this file from your PR.

**Review by boubkerbribri — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @nesrineabdmouleh

**Pull request merged**